de.haeckerfelix.gradio.json

```diff
@@ -0,0 +1,152 @@
+        "--talk-name=org.gnome.SessionManager",
+	"--filesystem=xdg-music:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O0 -g",
+        "cxxflags": "-O0 -g",
+        "env": {
+            "GST_PLUGIN_SYSTEM_PATH": "/app/lib/gstreamer-1.0/"
+        }
+    },
+    "modules": [
+        {
+            "name": "gstreamer",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gstreamer.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-base",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-plugins-base.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-good",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-plugins-good.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-ugly",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-plugins-ugly.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-bad",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-plugins-bad.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "yasm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
+                    "sha256": "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
+                }
+            ]
+        },
+        {
+            "name": "gst-libav",
+            "cleanup": [ "/bin/*", "/lib/peas-demo" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-libav.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "gst-devtools",
+            "config-opts": ["--validate", "--disable-gtk-doc"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-devtools.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "meson",
```

Comment:
Meson is already there so no need to add it

Reply:
Yeah you're right.